bootstrap-salt.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I guess the blank line here was added occasionally, right?

Reply:
Yes, I suppose so. At least, I did not intend to add it.